descartes_light/core/include/descartes_light/solvers/ladder_graph/ladder_graph.h

```diff
@@ -157,6 +158,68 @@ class LadderGraph
    */
   void clear();
 
+  void toDotGraph(std::string filepath, const std::vector<unsigned>& path = std::vector<unsigned>()) const
```

Comment:
Implementation should be moved to `ladder_graph.hpp` also if this is only to be used internally then the definition can also be removed from here.

Reply:
I guess it'll probably only be used internally, but I'm not sure yet how it will best fit to be called. I guess some parameter passed to the solve call that says whether or not to save a dot graph file and the location to save it to?